tests/frameworks/test_huggingface.py

```diff
@@ -125,8 +126,6 @@ def test_huggingface_integration(tmp_dir, model, args, data, mocker):
 
     logs, _ = parse_metrics(live)
 
-    assert len(logs) == 10
```

Comment:
This does not seem like a good test to have in dvclive since it relies on how many metrics hf automatically logs, including system metrics over which dvclive has no control.